packages/main/src/main.ts

```
@@ -76,6 +81,8 @@ export class Main implements IDisposable {
   }
 
   protected init(additionalData: AdditionalData): void {
+    this.#plugins.forEach(plugin => plugin.onBeforeReady?.());
```

Biome (2.5.8)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)